graphstorm-processing/docker/build_gsprocessing_image.sh

```diff
@@ -22,6 +22,7 @@ Available options:
 -p, --path          Path to graphstorm-processing directory, default is the current directory.
 -i, --image         Docker image name, default is 'graphstorm-processing'.
 -v, --version       Docker version tag, default is the library's current version (`poetry version --short`)
+-s, --suffix        Suffix for the image tag, can be used to push custom image tags. Default is "".
```

Comment:
Why we need a suffix?
Users can customize IMAGE_NAME, right?

Reply:
They can customize the name, but that would lead to the creation of another ECR repository. We want to give users the option to customize just the tag.